src/styles/SearchBar.css

```
@@ -0,0 +1,28 @@
+.input-search-bar {
+    width: 668px;
```

O que acha de deixar dois espaços de identação? Para deixar no padrão do ESlint? 😃 
```suggestion
  width: 668px;
```